fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -288,6 +288,8 @@ PERCENTILE: 'PERCENTILE';
 PERCENTILE_UNION: 'PERCENTILE_UNION';
 PLUGIN: 'PLUGIN';
 PLUGINS: 'PLUGINS';
+PIPE: 'PIPE';
+PIPES: 'PIPES';
```

Comment:
Add these two words into `nonReserved` in StarRocks.g4

Reply:
done